lib/gooddata/rest/connection.rb

```diff
@@ -597,6 +596,10 @@ def process_response(options = {}, &block)
         elsif response.code == 204
           result = nil
           GoodData.rest_logger.debug 'Response: 204 no content'
+        elsif response.code == 200 && content_type.nil? && response.body.empty?
```

Comment:
Refers to 'response' more than self (maybe move it to another class?) (FeatureEnvy)